### ISSUE TITLE
Liquibase export of Oracle RAW-columns fails

### ISSUE THREAD

**Issue by alf** [Wisser/Jailer#96]
### Description of the Issue

We're trying to subset an Oracle database where we use the RAW column type for storing GUIDs. This fails, as per the attached 
[error.log](https://github.com/Wisser/Jailer/files/9710536/error.log).

### Steps to Reproduce the Issue

Spin up an Oracle XE-database, for instance using the following command:

```
docker run --rm -p 1521:1521 -e ORACLE_RANDOM_PASSWORD=yes -e APP_USER=TEST -e APP_USER_PASSWORD=TEST gvenzl/oracle-xe:18-slim
```

Connect to the database, create a table with a row:

```
sql TEST/TEST@localhost:1521/XEPDB1 << EOF
create table f(c raw(8));
insert into f values ('aaaaaaaa');
commit
EOF
```

Connect with Jailer and try to export the table to Liquibase format.

### Fix

I'm able to export the data I need by applying the following patch, however this might break something else for all I know:

```
modified   src/main/engine/net/sf/jailer/liquibase/LiquibaseXMLTransformer.java
@@ -253,7 +253,6 @@ public class LiquibaseXMLTransformer extends AbstractResultSetReader {
 				break;
 				
 			case Types.BLOB:
-			case Types.VARBINARY:
 				count = entityGraph.incLobCount();
 				Blob blob = singleRow.getBlob(columncount);
 				int lengthb = (int) blob.length();
@@ -319,6 +318,7 @@ public class LiquibaseXMLTransformer extends AbstractResultSetReader {
 			case Types.NVARCHAR:
 			case Types.NCHAR:
 			case Types.LONGNVARCHAR:
+			case Types.VARBINARY:
 				attrcolumn=createAttribute(columnname,VALUE,singleRow.getString(columncount));
 				break;
```

**Comment by Wisser** [comment]
This issue has been fixed in release 12.7.2.

**Comment by alf** [comment]
Can confirm that this works for us. Thank you.
